### ISSUE TITLE
Types wrongly allow to pass readonly props to constructor

### ISSUE THREAD

**Issue by jonath92** [Gr3q/types-gjs#7]
It is a minor issue but worth to fix at some time in future: 

Currently the types don't prevent me to write something like this: 

```ts
const { Label } = imports.gi.St
const { Text } = imports.gi.Clutter

const label = new Label({
    visible: false,
    clutter_text: new Text({
        use_markup: true
    })
})
```
This is wrong because `clutter_text` is readonly. GJS is throwing the following error: `[radio@driglu4it]: Property StLabel.clutter-text is not writable`. 

I haven't tested if this also effects other classes but as the types are auto generated I guess this to be the case.

**Comment by Gr3q** [comment]
Yeah, I had the assumption readonly props can be modified at init, but it's probably not the case.

I can change it, but it would be nice to confirm it with other readonly props/classes
